Prace_domowe/Praca_domowa4/Wolny_Krzysztof/PD4.ipynb

```diff
@@ -0,0 +1,1967 @@
+{
```

Comment:
<p>robisz fit na teście, a to jest zbrodnia</p><p> </p><p>i serio surface bardziej potrzebuje skalowania niż construction_year?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>